react-frontend/src/index.css

```diff
@@ -1,7 +1,7 @@
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', 'Varela'
```

Comment:
Hm. If I understood the w3schools docs correctly, the first font in the `font-family` will be loaded. If that does not exist, it will try the next one. So this code will try to load `-apple-system` first?

I.e. this code should probably look like this instead to load Varela first (and remove some of the fonts):
```
font-family: 'Varela', 'Helvetica Neue', -apple-system, BlinkMacSystemFont
```

Does this change make the fonts look any different?

Reply:
I fixed this now!